Module-3 || DBMS Schema Design/Queries.txt

```diff
@@ -0,0 +1,10 @@
+select projects.ProjectID,projects.ProjectName,work.EmpID,EmpName,work.RoleId,RoleName from work,employees,projects,roles where work.ProjectID=1 and employees.EmpID=work.EmpID and projects.ProjectID=work.ProjectID and roles.RoleID=work.RoleId and roles.RoleName!='Manager' order by ProjectID;
```

Comment:
please indent these querues so they are readable and comment on top of each query what they are doing.